examples/nine-pt-stencil-solver/doc/builds-on

```diff
@@ -0,0 +1 @@
+simple-solver poisson-solver
```

Comment:
You can also add the three-pt-stencil solver here and for twenty-seven-pt-stencil solve as well.